csharp-advanced-topics/ConcurrentBagInCsharp/Tests/Tests/Tests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ConcurrentBag;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Tests
+{
+	delegate string PrintMessage(string text);
+	delegate T Print<T>(T param1);
+
+	[TestClass]
+	public class Tests
+	{
```

Comment:
In this class you've used tabs, in other spaces. Go through the project and align these to either tabs or spaces.